app/javascript/components/form/VuelidateFormComponent.js

```diff
@@ -1,67 +1,58 @@
-import { required, email, minLength, maxLength, sameAs, helpers } from 'vuelidate/lib/validators'
+import { required, email, minLength, maxLength, sameAs } from 'vuelidate/lib/validators';
 import {submitted} from '@modules/validators';
-import {onTransitionTriggered} from "@modules/transitions"
-import renderless from "@mixins/renderless";
+import {onTransitionTriggered} from '@modules/transitions';
+import renderless from '@mixins/renderless';
 
 /**
  * A component to wrap a validated form. Uses [Vuelidate]{@link https://monterail.github.io/vuelidate/} for validation
+ *
  * @module
  * @mixes renderless
- * @vue-data {Object} formData a JSON representation of the object being modified by the form
- * @vue-data {Object} toggles an object to track toggle states on the form
  * @vue-data {?module:components/stepper/FormStepComponent} stepper the stepper in this form, if there is one
- * @vue-data {Object} [submissionError={}] the errors returned from the last submission attempt
- * @vue-prop {String} validate a comma-separated string of fields to validate in the form
+ * @vue-data {Object} [submissionError={}] a mutatable copy of the errors returned from the last submission attempt
+ * @vue-data {Object} formData a mutatable copy of the object being modified by the form
+ * @vue-data {Object} toggles a mutatable copy of an object to track toggle states on the form
+ * @vue-prop {String} validate an object containing information on what types of validation, if any, each field in the form needs
  * @vue-prop {Object} startToggles the starting state of toggles on the form
+ * @vue-prop {Object} value the starting state of the value of the object modified by the form
+ * @vue-prop {Object} error the starting state of the errors from the last submission attempt
  * @vue-computed {Object} slotScope the scope to bind to the slot
  * @vue-computed {Object} $v the vuelidate object
  * @listens form.onsubmit
- * @listens from.ajax:error
+ * @listens form.ajax:error
  * @listens module:components/form/ToggleComponent~toggle-event
  */
 export default {
 	mixins: [renderless],
 	data: function() {
-		let name = this.name
-		let fd = null
-		if (name) {
-			fd = {}
-			fd[name] = gon.form_obj[name]
-		} else {
-			fd = gon.form_obj
-		}
-
-		let dt = {
-			formData: fd,
-			toggles: this.startToggles,
+		return {
 			stepper: null,
-			submissionError: gon.submissionError || {},
-			finalized: false
+			// need these because vue doesn't like it when you mutate props
+			submissionError: this.error,
+			formData: this.value,
+			toggles: this.startToggles,
 		};
-
-		// get rid of passwords
-		// TODO this should not be necessary. make the json work properly from the back end
```

Comment:
did you get rid of this because you made the json work properly?

Reply:
yes